functional-samples/sample.theme/README.md

```diff
@@ -0,0 +1,9 @@
+# Sample Chrome Theme
+
+This is a Chrome [theme](https://developer.chrome.com/docs/extensions/develop/ui/themes), that shows a number of color customization options.
+
+## Running this extension
+
+1. Clone this repository.
+1. Load this directory in Chrome as an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
+1. Reset your theme on the Chrome [Appearance settings panel](chrome://settings/appearance)
```

Comment:
We should probably show this link in plain text and tell users to copy it to the address bar, since you can't click links to Chrome pages?